daos/rinkeby/necdaocurvelabs.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "necDAO",
+  "Avatar": "0x331dDC7Ed27451b5B4d25A41e8dF53cFF456DcA9",
+  "DAOToken": "0x6939C5672b575C2fa4E3393D1706c7A097219805",
+  "Reputation": "0x814d49253d595Ec7341e7307d2CB4964Ab9655F6",
+  "Controller": "0x5bdD6bdb6D35920DC1DeD2101Cb9b1D39DCEa92D",
+  "Schemes": [
+    {
+      "name": "UniswapProxy",
+      "alias": "UniswapProxy",
+      "address": "0x50AE25C926176D71F41A27Fe97fa4A7c31207131",
```

Comment:
is this scheme can be verify that on [etherscan](https://rinkeby.etherscan.io/address/0x50AE25C926176D71F41A27Fe97fa4A7c31207131#code) ? it is missing the implementation of of it.  [see](https://rinkeby.etherscan.io/proxyContractChecker?a=0x50AE25C926176D71F41A27Fe97fa4A7c31207131)

Reply:
This contract is now verified and registered as Proxy so people can access its implementation. I am the current `admin` of this proxy. I'm not sure we're gonna stick to this proxy architecture for the mainnet release but the point here was to be able to incrementally update the proxy if requested by the client with no need to update the subgraph again [cause the indexed address would stay the same]. In case we use the same architecture on mainnet, of course, the admin rights would be granted to the DAO :)

Reply:
preferable to not have that as a proxy for production

Reply:
> preferable to not have that as a proxy for production

I am curious to understand why not?